packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -424,4 +441,57 @@ export class GCFBootstrapper {
       });
     }
   }
+
+  /*
+   * Setting the process.env.WEBHOOK_TMP environment variable indicates
+   * that the webhook payload should be written to a tmp file in Cloud
+   * Storage. This allows us to circumvent the 100kb limit on Cloud Tasks.
+   *
+   * @param body
+   */
+  private async maybeWriteBodyToTmp(body: string): Promise<string> {
+    // Cloud tasks has a maximum payload size of 100kb, if a webhook payload is
+    // approaching this size, then serialize it to a temporary folder:
+    const MAX_PAYLOAD_SIZE = 65000;
+    if (process.env.WEBHOOK_TMP && body.length > MAX_PAYLOAD_SIZE) {
+      const tmp = `${Date.now()}-${v4()}.txt`;
+      const bucket = storage.bucket(process.env.WEBHOOK_TMP);
+      const writeable = bucket.file(tmp).createWriteStream();
+      logger.info(`uploading payload to ${tmp}`);
+      writeable.write(body);
+      writeable.end();
+      await new Promise((resolve, reject) => {
+        writeable.on('error', reject);
+        writeable.on('finish', resolve);
+      });
+      return JSON.stringify({
+        tmpUrl: tmp,
+      });
+    } else {
+      return body;
+    }
+  }
+
+  /*
+   * If body has the key tmpUrl, download the original body from a temporary
+   * folder in Cloud Storage.
+   *
+   * @param body
+   */
+  private async maybeDownloadOriginalBody(payload: {
+    [key: string]: string;
+  }): Promise<object> {
+    if (payload.tmpUrl) {
+      if (!process.env.WEBHOOK_TMP) throw Error('no tmp directory configured');
+      const bucket = storage.bucket(process.env.WEBHOOK_TMP);
+      const file = bucket.file(payload.tmpUrl);
+      const readable = file.createReadStream();
+      const content = await getStream(readable);
+      await file.delete();
```

Comment:
Total nit, but maybe something we don't need to `await` and we could just let happen in a background?

Reply:
I actually landed on letting the bucket policy just cleanup files, because the delete makes it so we can't retry.